server/public/model/config.go

```diff
@@ -1285,6 +1285,21 @@ func NewLogSettings() *LogSettings {
 	return settings
 }
 
+func (s *LogSettings) isValid() *AppError {
```

Comment:
@hanzei It appears this change breaks the deprecated `AdvancedLoggingConfig` field which can contain a filespec.  E.g. 

```
    "LogSettings": {
        "EnableConsole": true,
        "ConsoleLevel": "DEBUG",
        "ConsoleJson": false,
        "EnableColor": true,
        "EnableFile": true,
        "FileLevel": "INFO",
        "FileJson": true,
        "FileLocation": "",
        "EnableWebhookDebugging": true,
        "EnableDiagnostics": true,
        "VerboseDiagnostics": false,
        "EnableSentry": true,
        "AdvancedLoggingJSON": {},
        "AdvancedLoggingConfig": "/home/dlauder/Desktop/Mattermost/AdvancedLogging/audit2.json",
        "MaxFieldSize": 2048
    },
```

The error on server start-up is:
`Error: failed to load configuration: failed to create store: unable to load on store creation: invalid config: LogSettings.isValid: Failed to parse JSON: invalid character '/' looking for beginning of value, invalid character '/' looking for beginning of value
`

Reply:
Good catch :+1:.  I filled https://mattermost.atlassian.net/browse/MM-55266 and submitted https://github.com/mattermost/mattermost/pull/25304 to fix the issue.